src/user/getUserPurchasedGames.ts

```diff
@@ -0,0 +1,63 @@
+// import { AuthorizationPayload, ProfileFromUserNameResponse } from "../models";
+
+import { AuthorizationPayload } from "../models";
+import { GetUserPurchasedGamesResponse } from "../models/user-purchased-games-response.model";
+import { call } from "../utils/call";
+import { USER_GAMES_BASE_URL } from "./USER_GAMES_BASE_URL";
+
+export type GetUserPurchasedGamesOptions = {
+  platform?: Array<"ps4" | "ps5">;
+  size?: number;
+  start?: number;
+  sortBy?: string;
+  sortDirection?: "asc" | "desc";
+  isActive?: boolean;
+  subscriptionService?: string;
+};
+
+const defaultOptions: GetUserPurchasedGamesOptions = {
+  platform: ["ps4", "ps5"],
+  size: 500,
+  start: 0,
+  sortBy: "ACTIVE_DATE",
+  sortDirection: "desc",
+  isActive: true,
+  subscriptionService: "NONE"
+};
+
+/**
+ * A call to this function will retrieve the list of games for the logged in user.
+ * If the user cannot be found (either due to non-existence or privacy settings),
+ * an error will be thrown.
+ *
+ * @param authorization An object containing your access token, typically retrieved with `exchangeCodeForAccessToken()`.
+ * @param options.platform An array containing the platforms to filter by. Defaults to `["ps4", "ps5"]`.
+ * @param options.size The number of results to return. Defaults to `500`.
+ * @param options.start The offset to start from. Defaults to `0`.
+ * @param options.sortBy The field to sort by. Defaults to `"ACTIVE_DATE"`.
+ * @param options.sortDirection The direction to sort by. Defaults to `"desc"`.
+ * @param options.isActive Whether to return active games only. Defaults to `true`.
+ * @param options.subscriptionService The subscription service to filter by. Defaults to `"NONE"`.
+ */
+export const getUserPurchasedGames = async (
```

Comment:
This is not currently being exported by its module. Consumers should be able to do a clean import like:

```ts
import { foo } from "psn-api";
```

However, I don't think that will work here because this function is not being exported by the module's _index.ts_ file.

Some consumers are doing:

```
import * as psnApi from "psn-api";
```

I think this import syntax may actually work for the new function, but I think this is probably an anti-pattern.

I have merged the latest `main` branch into this PR. It contains updates allowing you to run `yarn playground` for experimentation. Inside *src/__playground.ts* you'll find an import from the public-facing _index.ts_ file. We should be able to import all functions that are public-facing there.

Reply:
Thank you for the comment! I noticed that GraphQL queries are located in a different folder so I moved the `getPurchasedGames` to the `graphql` folder in order to co-locate the code.
I have also added my function to the module's `index.ts` file.

Thanks to that I was able to successfully import and work with the code I have developed.